### DISCUSSION TITLE
Mirth Connect API call errors in Dashboard after new install

### DISCUSSION THREAD

**Discussion by rjkroll** [nextgenhealthcare/connect#5378] [Q&A]
Hi,

Today we uninstalled our old Mirth 3.x and installed Mirth 4.1.    

We are able to login to the new instance, but every like 20 seconds in the Dashboard, we get the following error.   It looks like its trying to hit the Client API, but cannot.    When I try to browse to the client API URL  <server>:8443\api   we receive the normal Cert not valid warnings, and if we click next, it takes us to the Mirth API page, but it never loads.    I have no idea were to start investigating this.   If I run a Netstat, I see port 8443 and in the firewall, port 8443 is open.    Not sure what else to do or where to investigate.

Method failed: HTTP/1.1 500 Server Error
com.mirth.connect.client.core.ClientException: Method failed: HTTP/1.1 500 Server Error
at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:533)
at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:457)
.......
at java.lang.Thread.run(Unknown Source)
**Caused by: com.mirth.connect.client.core.EntityException: {
"servlet":"org.glassfish.jersey.servlet.ServletContainer-7e60e6bd",
"message":"Request failed.",
"url":"/api/4.1.0/channels/statuses/initial",
"status":"500"
}**
at com.mirth.connect.client.core.ServerConnection.handleResponse(ServerConnection.java:530)
... 29 more

**Comment by pacmano1** [comment]
Seems like a broken install.  

Try a clean install on a new box.

**Reply by rjkroll** [reply]
What's wierd, is that if I go to that URL from my local machine (laptop) I am able to view the API page and login and make requests no problem.

**Reply by pacmano1** [reply]
I assume you are going to `https://<server-ip>:8443/api`, your post above is mixing backslash with forward slashes

So one remote machine works differently than your another remote machine?

**Reply by rjkroll** [reply]
Yes, sorry.   I typed it wrong above.     On the local box that instance is installed on, if we try to go to that URL or click the API link in Mirth itself, it takes us to API page, warns about the certificate error, then if we click continue it takes us to the Mirth API page where we see the spinning loading icon, but it never loads API page.

If I go to a different machine, or even my laptop, then go to the same URL (replacing localhost with the servername), it still warns about the Cert, but when we click though, the page opens, loads, and I am able to login and just the different APIs listed on the page.

We dont even need to use the API page, but the fact that the Dashboard is trying to make calls to it uncesscully, and we cant even get it to display locally, are probably related reasons I am assuming.  

This box had a previous version of Mirth installed on it, which we uninstalled, but I am assuming that may be causing the issue.

Its a VM, so worst case scenario we can blow it away and rebuild it I guess.   Just wierd, havne't had this issue in 10 years or so using Mirth.

**Comment by jonbartels** [comment]
Are you using the default `admin` user and password?

You also need to look at your `mirth.log` and see what errors the server is throwing. The errors shown are client side errors.
